### DISCUSSION TITLE
How do I get the YOLO badge?

### DISCUSSION THREAD

**Discussion by Maher-Reven** [community/community#19319] [Other Feature Feedback, Questions, & Ideas] [answered]
![Screenshot 2022-06-24 at 02 17 01](https://user-images.githubusercontent.com/72787268/175435819-a14cf520-8f95-4b7c-9c69-e163c55dc1de.png)

**Comment by melvincwng** [comment, marked as answer]
@Maher-Reven, to get YOLO badge, you need to merge a PR directly into a public code repo without review.

Hope this clarifies.

Thanks and have a great day ahead 😀

**Reply by Maher-Reven** [reply]
thank you for sharing that though! 
wish you have same

**Reply by melvincwng** [reply]
@Maher-Reven, for more information, please refer to [this link](https://github.com/Schweinepriester/github-profile-achievements).

It states the possible achievements that can be obtained at this point in time.

**Reply by melvincwng** [reply]
@Maher-Reven, no problem!

Glad I managed to help you ^^

**Reply by Maher-Reven** [reply]
@melvincwng  🙏

**Reply by dnnr1** [reply]
thanks

**Reply by dnnr1** [reply]
https://github.com/Schweinepriester/github-profile-achievements

**Comment by Nik-08** [comment]
Working way. You just need to:

Open your repository;
Create new branch;
Do some updates;
Then, create pull request to main branch;
IMPORTANT! Add reviewer. To do this, you need to add access to repository someone, then add him as reviewer;
After this, just click merge as you do it always
Check your profile and see YOLO achievement.
